src/app/space/settings/collaborators/collaborators.component.ts

```diff
@@ -119,6 +130,27 @@ export class CollaboratorsComponent implements OnInit, OnDestroy {
     );
   }
 
+  assignUserRole(userId: string, roleName: string) {
+    // admin, contributor
+    this.subscriptions.add(
+      this.permissionService
+        .assignRole(this.context.space.id, roleName, [userId])
+        .subscribe(
+          () => {
+            if (roleName === 'admin') {
+              this.adminCollaborators.push(userId);
+            } else {
+              this.adminCollaborators.splice(this.adminCollaborators.indexOf(userId), 1);
```

Comment:
@rohitkrai03 this is valid if `indexOf` returns `-1`. You'll remove the last item in the array.

Reply:
Done